source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Benefit to oVirt users
+oVirt users may now analyze and see thier network topology via the Skydive UI. 
+Skydive allows the users to see the result of their configuration done via oVirt.
+They may capture traffic from VM1 to VM2 for example or monitor the traffic between two VMs or hosts. 
+Also they may generate some traffic between 2 running VMs on different hosts for example.
```

Comment:
does skydive itself allow you to generate the traffic?

Reply:
Yes

Reply:
Ok, so suggest changing "Also they may generate some traffic between 2 running VMs on different hosts for example" to:
"Skydive can generate traffic between 2 running VMs on different hosts and then analyze."

Reply:
done